<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WidgetToolBuildingPositionNew</class>
 <widget class="QWidget" name="WidgetToolBuildingPositionNew" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Gebäude</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Beschreibung</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QTextEdit" name="textEdit_Desc" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QGroupBox" name="groupBox_8" >
        <property name="title" >
         <string>Gebäude</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QListWidget" name="listWidget_Buildings" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Name</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLineEdit" name="lineEdit_Name" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string>Position</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="lineEdit_Y" />
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineEdit_X" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>X:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QGroupBox" name="groupBox_5" >
        <property name="title" >
         <string>Optionen</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QPushButton" name="pushButton_Delete" >
           <property name="toolTip" >
            <string>Löscht den Knoten</string>
           </property>
           <property name="statusTip" >
            <string>Löscht den Knoten</string>
           </property>
           <property name="whatsThis" >
            <string>Löscht den Knoten</string>
           </property>
           <property name="text" >
            <string>Löschen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QGroupBox" name="groupBox_6" >
        <property name="title" >
         <string>Höhe (m)</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLineEdit" name="lineEdit_Height" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QGroupBox" name="groupBox_7" >
        <property name="title" >
         <string>Rotation</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>5</number>
         </property>
         <item row="0" column="0" >
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Rotation" />
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
